### PR TITLE
Add release notes.

### PR DESCRIPTION
**Pull request by jcvernaleo**
.gitignore emacs temp files.

### CODE REVIEW

**Comment by jcvernaleo** [comment]
And the rendered version:
https://github.com/jcvernaleo/decred-release/blob/jcv_notes/release-notes.md

**Comment by marcopeereboom** [comment]
ok

File: release-notes.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#[v0.1.6](https://github.com/decred/decred-release/releases/tag/v0.1.6)
+
+## 2016-06-22
+
+This is the initial release of dcrinstaller, the simple installer and
```
Comment (jrick) [inline release-notes.md]: dcrinstall, not dcrinstaller

**Comment by marcopeereboom** [comment]
good catch!

File: release-notes.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+upgrader for the decred software.
+
+Binaries are provided for a number of platforms.
+
+For usage instructions and more information see
+[README.md](https://github.com/decred/decred-release/blob/master/cmd/dcrinstall/README.md).
+
+See manifest-v0.1.6.txt for sha256sums of the packages and
+manifest-v0.1.6.txt.asc to confirm those shas.
```
Comment (jrick) [inline release-notes.md:14]: but these files don't exist in this repo?

Comment (jrick) [inline release-notes.md:14]: ok nvm, they're in the github releases.

**Pull request merged**